doc/index.rst

```diff
@@ -181,6 +212,8 @@ Release Process
 Timeframe
 ---------
 
+.. todo:: Update with information about the cycle_build
+
 The timeframe for release will be defined per application, by the Developer(s) and Product Owner(s), and should part of the High-level lsst.io site.
```

Comment:
and should part of the High-level lsst.io site. -> and should be part of the high-level lsst.io site.